docs/features/software-catalog/descriptor-format.md

```diff
@@ -444,6 +446,22 @@ Apart from being a string, the software catalog leaves the format of this field
 open to implementers to choose. Most commonly, it is set to the ID or email of a
 group of people in an organizational structure.
 
+### `spec.system` [optional]
+
+Links the system that the component belongs to, e.g. `artist-engagement-portal`.
+This field is optional.
+
+The software catalog expects a single string that references the name of an
+entity of the `kind` `System`.
+
+### `spec.domain` [optional]
```

Comment:
Seems that only systems belong to domains, right? 

https://github.com/backstage/backstage/issues/390

Reply:
You mean that all components, APIs or resource implicitly inherit the domain of their system? That would imply that a component requires a system to be part of a domain. Is this intended?

Reply:
My interpretation of the model was that there would not be any inheriting as such; just that a component simply would not have a domain relation at all.

And nah, didn't mean to imply that any of these fields/relations are mandatory either way. But yes if you do want to be part of a domain, the model does seem to imply that you could only be that indirectly through a system.

Reply:
@mbruggmann opinions?

Reply:
Hot take: maybe exclude the component -> domain linkage for now unless we find an easy consensus, it's easier to add than to remove

Reply:
That's a good idea, let's extend that later

Reply:
No strong opinions, but yes, it seems like the easiest place to start would be to only have domains for systems. As in, a domain is a "collection of related systems".